Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'rubocop', '>= 1.0', '< 2.0'
```

Comment:
- [ ] Please add `gem 'rspec'` to the GemFile so the users are able to install your project using `bundle install` and test it locally.